### ISSUE TITLE
Getting wrong warning - Drupal core 8.5.10

### ISSUE THREAD

**Issue by mikkeschiren** [FriendsOfPHP/security-advisories#361]
With Drupal core 8.5.10 I get this:
```

Symfony Security Check Report
=============================
1 packages have known vulnerabilities.

drupal/core (8.5.10)
 * [CVE-2019-6338][]: Critical - Third Party Libraries
 * [CVE-2019-6339][]: Critical - Arbitrary PHP code execution

[CVE-2019-6338]: https://www.drupal.org/sa-core-2019-001
[CVE-2019-6339]: https://www.drupal.org/sa-core-2019-002
```

This is not correct - that is for Drupal core 8.5.8 or earlier.

**Comment by xabbuh** [comment]
should be fixed with #362

**Comment by ocramleznem** [comment]
Ouch. Sorry for that.
